packages/sdks-tests/src/e2e-tests/editing.spec.ts

```diff
@@ -330,4 +333,19 @@ test.describe('Visual Editing', () => {
       await page.frameLocator('iframe').getByText('coffee info: Another coffee brand.').waitFor();
     });
   });
+
+  test.describe('SDK', () => {
+    test('should inject correct SDK data into iframe', async ({ page, basePort, sdk }) => {
+      test.skip(excludeGen1(sdk));
+      const msgPromise = page.waitForEvent('console',  msg => msg.text().includes(SDK_INJECTED_MESSAGE_GEN_2));
```

Comment:
as discussed, let's improve this test so it:

- first checks for the event
- then checks individually for apiKey and model in separate `expect(msgText).toContain()` calls